playwright/e2e/suites/inventory.disable.ts

```diff
@@ -0,0 +1,48 @@
+import { test, expect } from '@playwright/test';
+
+const waitFor = async (page, selector, timeout = 3000) => {
+  // Seems weird to await in here with nothing else going on, but I am doing that explicitly so I can
+  // ditch the return and keep the signature Promise<void>
+  await page.waitForSelector(selector, { timeout });
+};
+
+//ADE not sure what this is for?
+const sleep = (ms) => new Promise((r) => setTimeout(r, ms));
```

Comment:
Probably not needed anymore. It was just for sleeping to let the page hydrate / stop re-rendering, a bandaid.